src/rust/engine/options/src/parse_tests.rs

```diff
@@ -25,7 +25,12 @@ fn test_parse_string_list_empty() {
     assert!(parse_string_list("").unwrap().is_empty());
 }
 
-fn list_edit<I: IntoIterator<Item = &'static str>>(
+#[test]
+fn test_parse_int_list_empty() {
```

Comment:
Each int_list test mirrors the string_list test immediately above it. Some of the string tests deal with quoting and escaping, so obviously those are not mirrored for the simpler case of integers.